packages/emitter-framework/src/python/components/black-formatter.tsx

```diff
@@ -0,0 +1,11 @@
+import { Children } from "@alloy-js/core";
+
+export interface BlackFormatterModel {
+  children?: Children;
+}
+
+/** Apply the Python formatter `black` to the child components. */
+export function BlackFormatter({ children }: BlackFormatterModel) {
+  // TODO: Implement!
```

Comment:
I don't think you can do this in the same way you did before, will be interesting to see how you want to hook this in.